server/src/main/java/org/elasticsearch/common/settings/LocallyMountedSecrets.java

```diff
@@ -93,6 +93,24 @@ public LocallyMountedSecrets(Environment environment) {
         this.secretsFile = secretsFilePath.toString();
     }
 
+    /**
+     * Resolve a secrets directory path given an environment
+     * @param environment Elasticsearch environment
+     * @return Secrets directory within an Elasticsearch environment
+     */
+    public static final Path resolveSecretsDir(Environment environment) {
```

Comment:
nit: static methods don't need to be final

Reply:
Good catch! And thanks.